### PR TITLE
Update swedish.xml

### PR DESCRIPTION
**Pull request by Regnander**
In this PR I...
* added the untranslated lines from the English file
* reworded the warning/error dialog messages to make the text flow better in Swedish
  * for example "Continue?" -> "Do you want to continue?"
* replaced the TAB indents with regular spaces to match the format of the English file
* trimmed some existing lines where the UI was truncating text
* made sure each menu item has a unique [mnemonic](https://en.wikipedia.org/wiki/Mnemonics_(keyboard)) in the `File`, `Edit` and `Search` menus
* used [Microsoft's Language Portal](https://www.microsoft.com/en-us/language) to make sure the translation is using recognizable and consistent terminology for the more technical terms

I should also mention that I took some liberties translating the color picker labels in `Preferences` > `Dark Mode` since they won't make any sense in Swedish when translated word-for-word. Some examples are "Top" = "Primary window color", "Menu hot track" = "Hover color" and "Active" = "Active elements".

### CODE REVIEW

File: PowerEditor/installer/nativeLang/swedish.xml [hunk 1]
```diff
@@ -1,1263 +1,1491 @@
...
+            <XpUpdaterProblem title="Uppdaterare för Notepad++" message="Uppdateraren för Notepad++ är inte kompatibel med XP på grund av ett föråldrat säkerhetslager i XP.
+Vill du gå till hemsidan för Notepad++ för att ladda ned den senaste versionen?"/>  <!-- HowToReproduce: Run menu "? -> Update Notepad++" under Windows XP. -->
+            <GUpProxyConfNeedAdminMode title="Proxyinställningar" message="Var god starta om Notepad++ som administratör för att konfigurera proxyn."/>
+            <DocTooDirtyToMonitor title="Övervakningsproblem" message="Dokumentet har ändrats. Spara ändringarna innan du övervakar den."/>
+            <DocNoExistToMonitor title="Övervakningsproblem" message="Filen måste finnas för att kunna övervakas."/>
+            <FileTooBigToOpen title="Problem med filstorleken" message="Filen är för stor för att öppnas med Notepad++"/> <!-- HowToReproduce: Try to open a 4GB file (it's not easy to reproduce, it depends on your system). -->
+            <CreateNewFileOrNot title="Skapa ny fil" message="&quot;$STR_REPLACE$&quot finns inte. Vill du skapa den?"/>
```
Comment (MarkusBodensee) [inline PowerEditor/installer/nativeLang/swedish.xml:1235]: Build failed due to xml error. Missing semicolon at second `&quot;`.

Comment (Regnander) [reply PowerEditor/installer/nativeLang/swedish.xml]: Done

**Comment by donho** [comment]
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#guidelines-for-pull-requests

**Comment by MarkusBodensee** [comment]
@Regnander I think you should avoid reformatting tabs to spaces to get a PR accepted.

**Comment by Regnander** [comment]
> @Regnander I think you should avoid reformatting tabs to spaces to get a PR accepted.

I haven't had the time to take a look on this until now, but thanks for the heads-up.

However, it seems like the commits I make to correct this won't show up in this PR.

**Pull request closed without merging**